agenda-connectors-api/src/main/java/org/exoplatform/agendaconnector/service/ExchangeConnectorService.java

```diff
@@ -59,4 +59,16 @@ List<EventEntity> getExchangeEvents(long userIdentityId,
                                       String start,
                                       String end,
                                       ZoneId userTimeZone) throws IllegalAccessException;
+
+
+  /**
+   * push event in exchange agenda.
+   *
+   * @param userTimeZone User time zone
+   * @param identityId User identity
+   * @param event Event to be pushed in exchange agenda
```

Comment:
```suggestion
   * @param event Event to be pushed to exchange agenda
```

Reply:
- Reorder java doc params for all methods
- Fix java doc params for getExchangeEvents method